packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/tooltip-rename-reference');
+
+ruleTester.run("tooltip-rename-reference", rule, {
+  valid: [
+    {
+      code: `import { Tooltip } from '@patternfly/react-core'; <Tooltip triggerRef />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Tooltip reference />`,
+    }
+  ],
+  invalid: [
+    {
+      code:   `import { Tooltip } from '@patternfly/react-core'; <Tooltip reference />`,
+      output: `import { Tooltip } from '@patternfly/react-core'; <Tooltip triggerRef />`,
+      errors: [{
+        message: `reference prop has been removed for Tooltip and replaced with the triggerRef prop.`,
```

Comment:
This wording needs to match that in the rule

Reply:
are you able to run tests locally?

Reply:
Yes, thanks! I forgot to run the tests. I wanted to specify a "better" info message but I did not change it in the tests. Should be fixed now (I kept the generated message)